sci-biology/bowtie/bowtie-2.2.4.ebuild

```diff
@@ -7,47 +7,46 @@ EAPI=5
 inherit eutils toolchain-funcs
 
 DESCRIPTION="An ultrafast memory-efficient short read aligner"
-HOMEPAGE="http://bowtie-bio.sourceforge.net/"
+HOMEPAGE="http://bowtie-bio.sourceforge.net/bowtie2/"
 SRC_URI="mirror://sourceforge/project/${PN}-bio/${PN}2/${PV}/${PN}2-${PV}-source.zip"
 
 LICENSE="GPL-3"
 SLOT="2"
-IUSE="examples"
 KEYWORDS="~amd64 ~x86"
 
+IUSE="examples cpu_flags_x86_sse2"
+REQUIRED_USE="cpu_flags_x86_sse2"
+
 RDEPEND="dev-lang/perl"
 DEPEND="${RDEPEND}
 		app-arch/unzip"
 
 S="${WORKDIR}/${PN}2-${PV}"
 
-pkg_pretend() {
-	grep "sse2" /proc/cpuinfo > /dev/null
-	if [[ $? -ne 0 ]] ; then
-		ewarn "Your processor does not support sse2. Bowtie will probably not work on this machine."
-	fi
-}
+PATCHES=( "${FILESDIR}"/${P}-buildsystem.patch )
 
 src_prepare() {
-	epatch "${FILESDIR}/${P}-buildsystem.patch"
+	epatch ${PATCHES[@]}
 }
 
 src_compile() {
+	unset CFLAGS
 	emake \
-		CC="$(tc-getCC)" \
-		CPP="$(tc-getCXX)" \
+		CXX="$(tc-getCXX)" \
 		EXTRA_FLAGS="${LDFLAGS}" \
 		RELEASE_FLAGS="${CXXFLAGS} -msse2"
 }
 
 src_install() {
 	dobin ${PN}2 ${PN}2-*
-	exeinto /usr/share/${PN}2/scripts
+
+	exeinto /usr/libexec/${PN}2
 	doexe scripts/*
 
-	newman MANUAL ${PN}2.1
+	newman MANUAL ${PN}2.2
```

Comment:
Hmm, manpages are special in the sense that the suffix determines the category. `2` stands for "system calls`, which doesn't seem right.